gloo/docs/installation/glooe.md

```diff
@@ -158,6 +158,16 @@ replicaset.apps/rate-limit-86b56f8c8b                            1         1
 replicaset.apps/redis-7f6954b84d                                 1         1         1       11m
 ```
 
+## Access the Gloo Enterprise UI
+
+The quickest way to access the Gloo Enterprise UI, that's portable to every kubernetes cluster, is
+via `kubectl port-foward`. To expose the UI, run this command:
+```
```

Comment:
add a new line between text and code fence - more consistent markdown behavior everywhere

change to ```bash as Github and Hugo will do better with code syntax highlighting